x/feedist/msg.go

```diff
@@ -0,0 +1,62 @@
+package feedist
+
+import (
+	"fmt"
+
+	"github.com/iov-one/weave/errors"
+)
+
+const (
+	pathNewRevenueMsg    = "feedist/newrevenue"
+	pathDistributeMsg    = "feedist/distribute"
+	pathUpdateRevenueMsg = "feedist/updateRevenue"
+)
+
+func (msg *NewRevenueMsg) Validate() error {
+	if err := msg.Admin.Validate(); err != nil {
+		return errors.Wrap(err, "invalid admin address")
+	}
+	if len(msg.Recipients) == 0 {
```

Comment:
q: Do we want to enforce a max number of recipient?

Reply:
Not that I know of.

Reply:
If so, make it quite big... Like 100.

But yeah, it can get quite expensive to distribute to many accounts at once. Gas price could be proportional to the number of recipients

Reply:
Only a maximum of 200 recipients is allowed. Implemented in https://github.com/iov-one/weave/pull/316/commits/7f699ed7f68580c219ff635f4cd6dc15f5c17b86